libtiledbsoma/test/test_soma_reader.py

```diff
@@ -78,6 +78,24 @@ def test_soma_reader_dim_points():
     assert sr.results_complete()
     assert arrow_table.num_rows == len(obs_id_points)
 
+def test_soma_reader_empty_dim_points():
```

Comment:
Nice -- as this is direct from my #593 this proves you're accomplishing the same thing I was